app/code/Magento/Ui/Model/Export/SearchResultIterator.php

```diff
@@ -22,40 +22,49 @@ public function __construct(
     }
 
     /**
+     * @inheritDoc
+     *
      * @return array|mixed
```

Comment:
if we use `@inheritdoc`, the following `return` annotation is redundant,  especially in the form of `array|mixed`

Reply:
Thanks, done